src/Symfony/Component/ObjectMapper/Exception/MappingTransformException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ObjectMapper\Exception;
+
+/**
+ * @experimental
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ */
+class MappingTransformException extends RuntimeException
```

Comment:
should this be final?

Reply:
I don't know the exact Symfony convention about this, but I would live it open, you might want in the future to create more specific exceptions extending this one, ie. "NonExistingPropertyException" or "NonObjectException" or whatsoever.

Reply:
If _we_ want to create child classes in the future, we might remove the `final` keyword in the future. This is a backward compatible change.